source/_integrations/event.markdown

```diff
@@ -43,7 +43,7 @@ Besides the timestamp of the last event, the event entity also keeps track of th
 
 This allows you, for example, to trigger a different action when the button on a remote control is pressed once or twice, if your remote control is capable of emitting these different types of events.
 
-When combining that with the [choose action](/docs/scripts/#choose-a-group-of-actions) script, you can assign multiple different actions to a single event entity. In the following example, pressing the button on the remote once or twice will trigger a different scene:
+When combining that with the [choose action](/docs/scripts/#choose-a-group-of-actions) script, you can assign multiple different actions to a single event entity. In the following example, pressing the button on the remote short or long will trigger a different scene:
```

Comment:
```suggestion
When combining that with the [choose action](/docs/scripts/#choose-a-group-of-actions) script, you can assign multiple different actions to a single event entity. In the following example, a different scene will be triggered depending on whether you press or press and hold the button:
```
The MS Style Guide seems to use _press and hold_ / _select and hold_
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/press-and-hold

also google:
https://support.google.com/accessibility/android/answer/7650693?hl=en

Alternatively, Apple use the term long-press:
https://developer.apple.com/documentation/uikit/touches_presses_and_gestures/handling_uikit_gestures/handling_long-press_gestures

Reply:
Not in this case, as it is a physical remote, and not an UI button.

Reply:
The style guide refers to a physical button.
![image](https://github.com/home-assistant/home-assistant.io/assets/38767475/ec5963da-702d-492c-883f-292774998e32)